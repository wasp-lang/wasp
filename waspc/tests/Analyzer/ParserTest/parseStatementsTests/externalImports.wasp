test Imports {
  module: import Page from "page.jsx",
  field: import { Page } from "page.jsx"
}
