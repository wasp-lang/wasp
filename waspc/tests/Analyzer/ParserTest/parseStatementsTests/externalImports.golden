(AST
  (Decl@1:1-4:1 type=test name=Imports
    (Dict@1:14-4:1
      (DictEntry key=module
        (ExtImport@2:11-37 module=Page path="page.jsx")
      )
      (DictEntry key=field
        (ExtImport@3:10-40 field=Page path="page.jsx")
      )
    )
  )
)
